src/gui/mzroll/background_peaks_update.cpp

```diff
@@ -485,54 +414,6 @@ void BackgroundPeakUpdate::align() {
                         aligner.doAlignment(groups);
                         mainwindow->sampleRtWidget->setDegreeMap(aligner.sampleDegree);
                         mainwindow->sampleRtWidget->setCoefficientMap(aligner.sampleCoefficient);
-                } else if (alignAlgo == 2) {
-                        mainwindow->getAnalytics()->hitEvent("Alignment", "LoessFit");
-                        aligner.preProcessing(groups, mavenParameters->alignWrtExpectedRt);
```

Comment:
`Aligner::preProcessing` is not being used anywhere. It was written to communicate data to the python executable from what I understand.

Reply:
Removed.